modules/importer.js

```diff
@@ -0,0 +1,46 @@
+import fs from "fs";
+
+export default class Importer {
+    constructor(eventEmitter, async = true){
```

Comment:
Please choose another name for the second parameter of constructor, since `async` is reserved keyword for asynchronous functions. E.g. `asyncMode` or something like that.

Reply:
Resolved